libraries/AP_BattMonitor/AP_BattMonitor_Params.h

```diff
@@ -27,6 +27,7 @@ class AP_BattMonitor_Params {
         BattMonitor_TYPE_FuelLevel_PWM              = 12,
         BattMonitor_TYPE_SUI3                       = 13,
         BattMonitor_TYPE_SUI6                       = 14,
+        BattMonitor_TYPE_DJIBattery                 = 15,
```

Comment:
small thing but I think we could probably remove the "Battery" suffix from the class name and this enum.  They're all batteries really so..